Pomodoro.Services/Realizations/StatisticsService.cs

```diff
@@ -102,7 +102,7 @@ public async Task<AnnualStatistics> GetAnnualStatisticsAsync(Guid userId, int ye
                 
                 foreach (var d in months)
                 {
-                    temp.PomodorosDone += d.PomodorosCount;
+                    temp.PomodorosDone += (int)d.PomodorosCount;
                 }
```

Comment:
Everything is fine but I think, it would be better for accuracy to calculate the sum at first and only then round float to the nearest integer. It can be smth like:

`float totalPomodorosCount = months.Sum(m => m.PomodorosCount);`
`temp.PomodorosDone = Convert.ToInt32(totalPomodorosCount);`

Reply:
It's a good idea. I'll do this for GetMonthlyStatisticsAsync and GetDailyStatisticsAsync too.